keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/NotificationEvent.kt

```diff
@@ -0,0 +1,13 @@
+package com.netflix.spinnaker.keel.events
+
+import com.netflix.spinnaker.keel.notifications.Notification
+import com.netflix.spinnaker.keel.notifications.NotificationScope
+import com.netflix.spinnaker.keel.notifications.NotificationType
+
+data class NotificationEvent(
+  val scope: NotificationScope,
+  val identifier: String,
+  val notificationType: NotificationType,
+  val message: Notification
```

Comment:
This is what I meant when I brought up encapsulation: these 3 fields could be collapsed into one, the `Notification` object, because those are part of what that object is intended to represent (the notification doesn't make sense without an associated type and identifier).